service/gcs/core/gcs/storage.go

```diff
@@ -177,6 +181,51 @@ func (c *gcsCore) unmountMappedVirtualDisks(disks []prot.MappedVirtualDisk) erro
 	return nil
 }
 
+// mountMappedDirectories mounts the given mapped directories using a Plan9
+// filesystem with the given options.
+func (c *gcsCore) mountMappedDirectories(dirs []prot.MappedDirectory) error {
+	for _, dir := range dirs {
```

Comment:
Do we need any cleanup logic here? I guess not really since the container is destroyed yes?